cli/search/search.go

```diff
@@ -20,6 +20,11 @@ const (
 	GitRepoTT        = "https://github.com/tarantool/tt.git"
 )
 
+const (
+	VersionCliSeparator = "="
```

Comment:
Please add comments for these constants.

Reply:
Added